extensions/applicationinsights-osplugin-js/src/DataModels.ts

```diff
@@ -0,0 +1,11 @@
+/**
+* DataModels.ts
+* @author Siyu Niu (siyuniu)
+* @copyright Microsoft 2024
+* File containing the interfaces for OS Plugin SDK.
+*/
+
+export interface IOSPluginConfiguration {
```

Comment:
Please also add the typedoc for this, as new components we should start by adding the documentation. Rather than requiring that we defer and add it later as part of the bigger typedoc improvements.